common/src/main/java/io/pravega/common/io/filesystem/FileOperations.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.common.io.filesystem;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+
+@Slf4j
+public class FileOperations {
+
+    public static void cleanupDirectories(HashMap<?, File> toDelete) throws IOException {
```

Comment:
Some context: a way was needed to add coverage to the catch block added in `cleanupDirectories`, but moved somewhat nuanced. We needed a platform independent way to force an `IOException`. Given the attachment to the `BookKeeperServiceRunner` we were quite limited in what we could do.

Refactoring this method out was not ideal, but it was motivated by the following:
* It allowed us to avoid modifying any of the visibility rules of field/methods under `BookKeeperServiceRunner`. 
* It also allows us to avoid adding a completely contrived and seemingly random test under `BookKeeperServiceRunnerTests`.